### ISSUE TITLE
Slide-panel doesn't switch back to aria-hidden="true" when closed

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2070]
# Bug Report

## Description
When slide-panel is initially closed, it sets the panel content to aria-hidden="true". When opened, it switches this to aria-hidden="false", when closed again, it fails to switch aria-hidden back to being "true".

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-ui/#/components/terra-slide-panel/slide-panel/slide-panel
2. Inspect DOM
3. Open slide panel in the first example on the page
4. Close the slide panel in the first example on the page
5. Inspect aria-hidden attribute

## Expected Behavior
Aria-hidden should properly reflect the state of the slide panel

## Environment

* Component Name and Version: slide-panel
* Browser Name and Version: All
